crates/experimentation_platform/src/api/experiments/types.rs

```diff
@@ -165,14 +163,34 @@ pub struct StatusTypes(
     pub  Vec<ExperimentStatusType>,
 );
 
+#[derive(Deserialize, Debug, Clone)]
+#[serde(rename_all = "snake_case")]
+pub enum ExperimentSortOn {
+    LastModifiedAt,
+    CreatedAt,
+}
+
+impl Default for ExperimentSortOn {
+    fn default() -> Self {
+        Self::LastModifiedAt
+    }
+}
+
 #[derive(Deserialize, Debug)]
-pub struct ExpListFilters {
+pub struct ExperimentListFilters {
     pub status: Option<StatusTypes>,
     pub from_date: Option<DateTime<Utc>>,
     pub to_date: Option<DateTime<Utc>>,
+    pub experiment_name: Option<String>,
+    pub experiment_ids: Option<StringArgs>,
+    pub created_by: Option<StringArgs>,
+    pub context: Option<String>,
+    pub sort_on: Option<ExperimentSortOn>,
+    pub sort_by: Option<SortBy>,
```

Comment:
```suggestion
    pub order: Option<Order>,
```

Reply:
No

Reply:
Isn't `Asc` and `Desc` an order in which the data has to be sorted, `sort_by` sounds more like on what you want to sort on

Reply:
I went with the way the operation is spoken about, you sort by descending order. You can also order by descending order, but the meaning is the same, so making the change is irrelevant